landbosse/excelio/XlsxParallelManagerRunner.py

```diff
@@ -49,14 +49,14 @@ def run_from_project_list_xlsx(self, projects_xlsx, log):
         """
         # Load the project list
         projects = pd.read_excel(projects_xlsx, 'Sheet1')
-        log.debug('>>> Project list loaded')
+        # log.debug('>>> Project list loaded')
 
         # Prep all task for the executor
         all_tasks = []
         for _, project_series in projects.iterrows():
-            project_id = project_series['Project ID']
+            project_data_basename = project_series['Project data file']
             task = dict()
-            task['input_xlsx'] = os.path.join(landbosse_input_dir(), 'project_data', '{}.xlsx'.format(project_id))
+            task['project_data_xlsx'] = os.path.join(landbosse_input_dir(), 'project_data', f'{project_data_basename}.xlsx')
```

Comment:
Here is where I get an independently named `Project data file` for each project.